fast64.json

```diff
@@ -0,0 +1,28 @@
+{
+  "version": 1.0,
+  "autoLoad": true,
+  "microcode": "F3DEX2/LX2",
+  "rdpDefaults": {
+    "geometryMode": {
+      "zBuffer": true,
+      "shade": true,
+      "cullBack": true,
+      "lighting": true,
+      "shadeSmooth": true
+    },
+    "otherModeH": {
+      "textureConvert": "G_TC_FILT",
+      "textureFilter": "G_TF_BILERP",
+      "perspectiveCorrection": "G_TP_PERSP",
+      "pipelineMode": "G_PM_1PRIMITIVE"
+    },
+    "otherModeL": {},
+    "other": {}
+  },
+  "sm64": {
+    "refresh_version": "Refresh 13",
+    "compression_format": "mio0",
```

Comment:
This should be "yay0" I assume given all load commands look like `LOAD_YAY0`.

Ideally of course it shouldn't include the name of the compression algorithm at all and look like `LOAD_COMPRESSED` but that is out of the scope of this PR.